lang/cs/Org.Apache.REEF.IMRU/API/UpdateResult.cs

```diff
@@ -50,22 +50,34 @@ internal bool IsNotDone
             get { return !_done; }
         }
 
-        internal bool HasMapInput
+        /// <summary>
+        /// Indicates if the UpdateResult has map input data
+        /// </summary>
+        public bool HasMapInput
```

Comment:
UpdateResult is defined in IMRU and used in IMRU.Example in task functions.  Since UpdateResult is exposed to the IMRU example as public, its property can be public as well.